src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -299,9 +308,11 @@ static SuggestedActionSetPriority ConvertToSuggestedActionSetPriority(UnifiedSug
                 // See https://github.com/dotnet/roslyn/issues/29589
                 const bool includeSuppressionFixes = true;
 
-                return UnifiedSuggestedActionsSource.GetFilterAndOrderCodeFixesAsync(
+#pragma warning disable CA2007 // Consider calling ConfigureAwait on the awaited task (this method manages affinity using JTF)
```

Comment:
>#pragma warning disable CA2007  [](start = 0, length = 31)

Having these suppressions is unfortunate - it clutters the code.

Reply:
I need to work with @mavasani to update the analyzer.

Reply:
@sharwell Can you elaborate more on when the analyzer shouldn't report diagnostic?
I may work on that and open a PR against roslyn-analyzers.

Reply:
`ConfigureAwait` and JTF offer two different approaches to the same problem. While it's not necessarily a _bug_ to use `ConfigureAwait` after a JTF affinity operation, it's generally preferred to only explicitly state affinity at points where affinity matters for the operation.

Reply:
@sharwell So, basically if the await is inside the lambda passed to ThreadingContext.JoinableTaskFactory.Run, there is no need to report a diagnostic, right?

Reply:
Yes, plus it would cover cases where the `await` comes after one of these:

```
await JTF.SwitchToMainThreadAsync();
```

```
await TaskScheduler.Default;
```

Reply:
IDE policy here has been to be explicit about using .ConfigureAwait(true) for this.  It's idiomatic and clear as to the intention.  It's what we use elsewhere.  I would prefer we continue to stick with that.

Reply:
> It's idiomatic and clear as to the intention. 

This is true for some scenarios, but not for JTF scenarios. The analyzer updates will help clean this up by keeping each method idiomatic according to the strategy it uses.

Reply:
i am amenable to that *once* we have those "analyzer updates" :)  at that point, then i don't ahve to think about it.  right now i'd like to stay in teh "we don't have to think about it because everything is consistent" camp :)

Reply:
note: i'm not dying on this hill.  if anything, take this to mean that i would strongly encourage prioritization of those updates so that we can both have what we want here :)

Reply:
for now we can resolve this bit of feedback as acceptable.